tm2/pkg/colors/colors.go

```diff
@@ -107,3 +107,15 @@ func ColoredBytes(data []byte, textColor, bytesColor func(...interface{}) string
 	}
 	return s
 }
+
+func ColoredBytesOnlyAscii(data []byte, textColor func(...interface{}) string) string {
+	s := ""
+	for _, b := range data {
+		if 0x21 <= b && b < 0x7F {
```

Comment:
What do you think about defining a method called `isASCII` that does this check on a single byte?

Reply:
875394f02faf82f6e2f4b1136d1be3ae3666f257